events/views.py

```diff
@@ -67,32 +71,36 @@ def post(self,request,id):
         return Response(responses.user_unauthorized_401,status.HTTP_401_UNAUTHORIZED)
 
 # TODO: simplify with drf
-@api_view(['POST', ])
-@ecell_user
-def event_unregister(request, id):
-    u = request.ecelluser
-    if u:
-        try:
-            e = Event.objects.get(id=id)
-        except:
-            res_message="Event does not exist"
-            res_status=status.HTTP_404_NOT_FOUND   
-        else:
+class EventUnregisterView(APIView):
+    permission_classes = [IsAuthenticated]
+    @swagger_auto_schema(
+        operation_id="eventunregister",
+        responses={
+            '200' : set_example(responses.event_registration_deleted_200),
+            '403' : set_example(responses.user_forbidden_403),
+            '404' : set_example(responses.event_does_not_exist_404,responses.event_not_registered_404)
```

Comment:
1. The `set_example` method in `utils/swagger.py` accepts just one `example` as argument.
2. Add `401` to responses.

Reply:
Done!